README.md

```diff
@@ -1 +1,156 @@
-# country-global
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+## 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+   - [Usage](#usage)
+- [👥 Authors](#authors)
+ - [🚀 Live Demo](#-live-demo-)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 SPACE TRAVELERS HUB <a name="Portfolio-site"></a>
+
+**Space Traveler's Hub** is an app for Space X that provides commercial and scientific space travel services. Built with React and Redux, The app allows users to reserve rockets and join selected space missions.
+
```

Comment:
- [ ] Kindly pay attention to your README documentation. This is a capstone project, but you have a description for Space Travellers group project. Kindly fix this.